src/webgpu/util/conversion.ts

```diff
@@ -18,24 +18,40 @@ import {
 } from './math.js';
 
 /**
- * Encodes a JS `number` into a "normalized" (unorm/snorm) integer representation with `bits` bits.
- * Input must be between -1 and 1 if signed, or 0 and 1 if unsigned.
+ * Encodes a JS `number` into "normalized" (unorm/snorm) integer scale with `bits` bits for ULP
+ * comparison. Input must be between -1 and 1 if signed, or 0 and 1 if unsigned.
+ * e.g. float 0.5 -> "unorm8" 127.5
  *
  * MAINTENANCE_TODO: See if performance of texel_data improves if this function is pre-specialized
  * for a particular `bits`/`signed`.
  */
-export function floatAsNormalizedInteger(float: number, bits: number, signed: boolean): number {
+export function floatAsNormalizedIntegerScale(
```

Comment:
Scale -> Unquantized maybe? And explain that this doesn't cut off the fractional part of what would otherwise be an integer so that additional computations can be performed?